yarn.lock

```diff
@@ -1637,19 +1637,19 @@
   resolved "https://registry.yarnpkg.com/@guardian/prettier/-/prettier-0.6.0.tgz#b3dcbef0e2a9232d2afe04dff075426b2939f44e"
   integrity sha512-RN8GBRth5BXHoWymCZIYDbmZM8DkUbgecf+wyv+9hdXA/ad4Ob4qP9u/ARU689qMo+JSC1z3FRKelvkkjtYzJw==
 
-"@guardian/src-brand@^3.3.0":
+"@guardian/src-brand@3.8.0":
```

Comment:
I guess we forgot to commit the lock file with the banner fix PR yesterday? 😂

Reply:
yep!